docs/src/content/docs/guides/split-payments.mdx

```diff
@@ -0,0 +1,331 @@
+---
+title: Split payments
+---
+
+import { CodeBlock, LinkOut } from '@interledger/docs-design-system'
+import { Tabs, TabItem } from '@astrojs/starlight/components'
+import HttpSig from '/src/partials/http-msg-sig-note.mdx'
+import ChunkedSnippet from '/src/components/ChunkedSnippet.astro'
+import Ts from '/src/partials/ts-prerequisites.mdx'
+
+# Build an e-Commerce platform with split-payments between merchants and platform
+
+This tutorial guides you through building an e-Commerce platform using Open Payments where split-payments are initiated by a buyer of a good or service and split between the merchant and you, the platform operator. Airbnb and Uber are popular platforms where payments made by guests and riders are split between the platform on one side and hosts and drivers on the other respectively.
+
+## Dependencies
+
+- <LinkOut href='https://github.com/interledger/open-payments/tree/main/packages/open-payments'>
+    Open Payments SDK
+  </LinkOut>
+
+<HttpSig />
+
+## Endpoints
+
+- <LinkOut href='https://openpayments.dev/apis/auth-server/operations/post-request/'>
+    Grant Request
+  </LinkOut>
+- <LinkOut href='https://openpayments.dev/apis/wallet-address-server/operations/get-wallet-address/'>
+    Get a Wallet address
+  </LinkOut>
+- <LinkOut href='https://openpayments.dev/apis/resource-server/operations/create-incoming-payment/'>
+    Create an Incoming Payment
+  </LinkOut>
+- <LinkOut href='https://openpayments.dev/apis/resource-server/operations/create-quote/'>
+    Create a Quote
+  </LinkOut>
+- <LinkOut href='https://openpayments.dev/apis/resource-server/operations/create-outgoing-payment/'>
+    Create an Outgoing Payment
+  </LinkOut>
+
+<Tabs>
+<TabItem label='TypeScript'>
+<Ts />
+
+## Steps
+
+### 1. Import dependencies
+
+Import `createAuthenticatedClient` from the Open Payments SDK package.
+
+<ChunkedSnippet
+  source='https://raw.githubusercontent.com/interledger/open-payments-snippets/main/incoming-payment/incoming-payment-create.ts'
+  chunk='1'
+/>
+
+### 2. Create an authenticated Open Payments client
+
+Create an Open Payments-authenticated client by providing the following properties:
+
+- `walletAddressURL` : your Open Payments-enabled wallet address that your client will use to authenticate itself to one or more Authorization Servers (AS).
```

Comment:
I know it's mentioned in the overview, but I think it would be good to reiterate who you/your is (are?).